test/util/connectivity/connectivity.go

```diff
@@ -0,0 +1,57 @@
+package connectivity
+
+import "fmt"
+
+// ConnectivityTest .
+type ConnectivityTest struct {
```

Comment:
I would keep this and `Parameter` type safe and change only the rendering. It will make everything else easier.
So, a struct that contains the exact stuff you want (protocol, mtu, connectivity) and a way to render it in a format you like.